### PR TITLE
Docs troubleshooting

### PR DESCRIPTION
**Pull request by ahgittin**
Fix merge conflicts from #765 (moving @nakomis 's "exceptions" troubleshooting) and attempt to update titles so menu is more intuitive.  @aledsage feedback welcome.  BTW merging this should close #765 due to the commit history.

### CODE REVIEW

**Comment by ahgittin** [comment]
Updated following #766 merger.  For the avoidance of doubt probably manually close #765.

File: docs/guide/ops/troubleshooting/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Troubleshooting
+layout: website-normal
+children:
+<<<<<<< HEAD
```
Comment (aledsage) [inline docs/guide/ops/troubleshooting/index.md]: Merge conflict text has sneaked in.

**Comment by aledsage** [comment]
Thanks @ahgittin - changes make sense. Only one minor thing where merge conflict text has sneaked into file. Once that's fixed, please do merge!

**Comment by hzbarcea** [comment]
+1

**Comment by aledsage** [comment]
Thanks @ahgittin - merging.

**Pull request merged**